modelskill/types.py

```diff
@@ -25,17 +25,17 @@ def from_string(s: str) -> "GeometryType":
         Examples
         --------
         >>> GeometryType.from_string("point")
-        <GeometryType.POINT: 1>
+        <GeometryType.POINT: 'point'>
         >>> GeometryType.from_string("track")
-        <GeometryType.TRACK: 2>
+        <GeometryType.TRACK: 'track'>
         >>> GeometryType.from_string("unstructured")
-        <GeometryType.UNSTRUCTURED: 3>
+        <GeometryType.UNSTRUCTURED: 'unstructured'>
         >>> GeometryType.from_string("flexible mesh")
-        <GeometryType.UNSTRUCTURED: 3>
+        <GeometryType.UNSTRUCTURED: 'unstructured'>
         >>> GeometryType.from_string("dfsu")
-        <GeometryType.UNSTRUCTURED: 3>
+        GeometryType.from_string("flexible mesh")
```

Comment:
<GeometryType.UNSTRUCTURED: 'unstructured'> ?

Reply:
😬 which is why executable examples are preferable. See #421